common/remote/sockfile.cpp

```diff
@@ -3987,6 +3993,23 @@ class CRemoteDiskReadActivity : public CSimpleInterfaceOf<IRemoteActivity>
     {
         return outputGrouped;
     }
+
+ //interface IVirtualFieldCallback
+    virtual const char * queryLogicalFilename(const void * row)
+    {
+        const char * filename = nullptr;
+        return filename ? filename : "";
+    }
+    virtual unsigned __int64 getFilePosition(const void * row)
+    {
+        unsigned __int64 baseOffset = 0;
+        return prefetchBuffer.tell() + baseOffset;
+    }
+    virtual unsigned __int64 getLocalFilePosition(const void * row)
+    {
+        unsigned part = 0;
+        return makeLocalFposOffset(part, prefetchBuffer.tell());
```

Comment:
Because 0 is wrong - it needs to be passed from thor to dafilesrv and then correctly used in this call.  Should have a comment/jira for the changes that need making in dafilesrv code.

Reply:
I have added some comments.